src/siteconfig/tests/test_forms.py

```diff
@@ -0,0 +1,91 @@
+from io import BytesIO
+
+from django.core.files.uploadedfile import InMemoryUploadedFile
+
+from django_tenants.test.cases import TenantTestCase
+
+from siteconfig.models import SiteConfig
+from siteconfig.forms import SiteConfigForm
+
+from hackerspace_online.tests.utils import model_to_form_data
+
+
+class SiteConfigFormTest(TenantTestCase):
+    """Tests for the SiteConfig Form
```

Comment:
Currently, it will not let me clear the files:
![Peek 2023-05-24 21-12](https://github.com/bytedeck/bytedeck/assets/10604391/2573726f-7ee5-4055-a667-4b80a3851dac)

Reply:
see latest commit